### DISCUSSION TITLE
Transcription response is empty

### DISCUSSION THREAD

**Discussion by osamabinsaleem** [deepgram/community#765] [General help]
### Which Deepgram product feature are you using?

Deepgram SDKs

### Details

for this particular audio, this is the repsone that we get: 
```

"alternatives": [
{
"transcript": "",
"confidence": 0.0,
"words": []
}
],

```
and when I used the same audio on the Deepgram playground, I got the accurate trasnscript. This is not awlays happening but soemtimes it does meaning that the code is fine. I get the same results when the audio is muted but that's expected. But for an audio with actual speech content, transcriptions should always be there and if Deepgram gets any issues those issues should be returned as error responses. But in this case, the audio file was valid but the transcript was still empty. 

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

8df5b9ee-4a2b-43c2-8a79-544ed391e56d

### If possible, please attach your code or paste it into the text box.

```
with open(audio_efs_path, "rb") as file:
            buffer_data = file.read()

        payload: FileSource = {
            "buffer": buffer_data,
        }

options = PrerecordedOptions(
                    model="nova-2-general",
                    smart_format=True,
                    utterances=True,
                    detect_language=True,
                    keywords=deepgram_format_keywords,
                )

response = deepgram.listen.prerecorded.v("1").transcribe_file(
            payload, options, timeout=httpx.Timeout(300.0, connect=10.0)
        )
```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by davidvonthenen** [reply]
What version of the SDK are you using?
Have you tried one of the examples in the repo to make sure everything works?
Have been able to try other files or does it always come without a transcription regardless of the file?
